tests/cli/test_utils.py

```diff
@@ -26,3 +27,7 @@ def test_yn():
             inp.send_text("n")
             result = yn("????")
             assert result == False
+
+
+def test_hostname():
+    assert get_fully_qualified_name() == socket.getfqdn()
```

Comment:
```suggestion
    assert get_fully_qualified_name()
```

These are not equal on my Mac, for reasons that I consider unimportant because all that matters is that it contains `server.mila.quebec` on the Mila cluster, which is the case. So I'd just make sure it returns something.